packages/common/src/store/pages/token-dashboard/types.ts

```diff
@@ -14,6 +14,8 @@ type SendingState =
       amount: StringWei
       recipientWallet: string
       chain: Chain
+      canRecipientReceiveWAudio: boolean
+      loading: false
```

Comment:
Thinking through the data modeling here - rather than modeling this as two separate booleans, where there are certain combos that don't make sense (canReceive = false, loading = true), and it's not super clear what loading refers to, I think we could simplify this by giving canReceiveWaudio 3 states: `'false' | 'loading' | 'true'`